### DISCUSSION TITLE
EEPROM write cycles

### DISCUSSION THREAD

**Discussion by optimismus** [cbrosius/luxtronik_v1#23] [Ideas]
Hi, during a discussion [Simon42](https://community.simon42.com/t/waermepumpensteuerung-basierten-auf-guenstigem-strom/16389/1) I got the information, that the EEPROM has limited write cycles. That is not a totally new information, but I didn't reconize this so far.
We have to avoid to destroy the EEPROM for an engery optimization, I think.

**Comment by cbrosius** [comment]
Hier ein Link zur Umfrage, ob ich das Schreiben von Werten einbauen soll: https://github.com/cbrosius/luxtronik_v1/discussions/30
